apps/sms/views/conversation/js/conversation.js

```diff
@@ -2009,7 +2077,8 @@ var ConversationView = {
     var bubble;
 
     do {
-      if (node.classList.contains('bubble')) {
+      // A Text node does not have a classList property
+      if (node.classList && node.classList.contains('bubble')) {
```

Comment:
not a big deal, I just see this change now; did you have any issue about it ?  I think "getMessageBubble" is only called in one place, and with evt.target as parameter I think it's always an element ?